install_config/advanced_ldap_configuration/sssd_for_ldap_failover.adoc

```diff
@@ -67,6 +67,15 @@ in this topic they are kept separate.
 [[sssd-phase-1-certificate-generation]]
 == Phase 1: Certificate Generation
 
+[NOTE]
+====
+The undermentioned commands generate certificate files that will be valid for 2
+years (and 5 years for Certification authority (CA) certificate). These
+periods can be altered with `--expire-days` and `--signer-expire-days` options
```

Comment:
"Extending" means that we can only make the value greater, but this option allows to **change** the value and user decides whether make it greater or lesser.